.github/workflows/labeler.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (C) 2022-2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: "PR Labeler"
+
+on:
+  - pull_request_target
```

Comment:
I suspect some events are missing

Reply:
related to https://github.com/podman-desktop/podman-desktop/blob/main/.github/workflows/pr-check.yaml#L23

default one are ok for this usecase ?